apps/workplace/src/app/book/meeting-flow/meeting-flow-form.component.ts

```diff
@@ -464,6 +472,117 @@ export class MeetingFlowFormComponent extends AsyncHandler {
         return count;
     }
 
+    private _space_list = new BehaviorSubject<Space[]>([]);
+
+    private _assets_available = this._space_list.pipe(
+        debounceTime(300),
+        switchMap((space_list) => {
+            if (!space_list?.length) return of(false);
+            const value = this.form.getRawValue();
+            this._assets.setOptions({
+                date: value.date,
+                duration: value.duration,
+                resources: space_list,
+                zone_id: this._org.levelWithID(space_list[0].zones)?.parent_id,
+                tags: [],
+                categories: [],
+            } as any);
+            return combineLatest([
+                this._assets.filtered_assets,
+                this._assets.disabled_rooms,
+            ]).pipe(
+                map(([items, disabled_rooms]) => {
+                    const can_cater = space_list.every(
+                        (s) =>
+                            items.filter(
+                                (_) =>
+                                    !(_ as any).hide_for_zones?.find((z) =>
+                                        s.zones.includes(z)
+                                    )
+                            ).length > 0
+                    );
+                    if (
+                        can_cater &&
```

Comment:
Hey @MrYuion is this catering car relevant inside this assets function? Or is it just a leftover from copy/paste that shouldn't be here?